src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -118,7 +118,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       RestoreIgnoreFailedSources="$(RestoreIgnoreFailedSources)"
       RestoreRecursive="$(RestoreRecursive)"
       RestoreForce="$(RestoreForce)"
-      HideWarningsAndErrors="$(HideWarningsAndErrors)"/>
+      HideWarningsAndErrors="$(HideWarningsAndErrors)"
+      NuGetInteractive="$(NuGetInteractive)"/>
```

Comment:
I would have the _task_ property be `Interactive`

`Interactive="$(NuGetInteractive)"`